oci/oci-how-to/create-chiselled-ubuntu-image.rst

```diff
@@ -1,7 +1,7 @@
 Create a chiselled Ubuntu base image for C/C++, Go and Rust applications
```

Comment:
I think generally I'm a bit confused by the use of the word "base". It's not clear to me whether we mean Ubuntu Base images, or basic Ubuntu chiselled images. It would be good to clarify this and standardise on one term throughout.

Reply:
the term "base" here is misused. I'm removing it.